coalib/misc/Constants.py

```diff
@@ -109,7 +109,14 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+# Checks for default_coafile in coala/coalib
+for root, dirs, file_lists in os.walk('coalib'):
+    for file in file_lists:
+        if file == 'default_coafile':
+            logging.warning
```

Comment:
@sils Okay i will improve this but do i also need to add some test for this??? As @Techievena says being a newcomer issue it should not involve any tests writing .